src/main/controlMod.F90

```diff
@@ -1101,6 +1105,8 @@ subroutine control_print ()
        write(iulog, *) '    use_fates_fixed_biogeog = ', use_fates_fixed_biogeog
        write(iulog, *) '    use_fates_nocomp = ', use_fates_nocomp
        write(iulog, *) '    use_fates_sp = ', use_fates_sp
+       write(iulog, *) '    use_fates_luh= ', use_fates_sp
+       write(iulog, *) '    fluh_timeseries = ',fluh_timeseries
        write(iulog, *) '    fates_inventory_ctrl_filename = ',fates_inventory_ctrl_filename
```

Comment:
This isn't yours but could use a trim function around it.

Reply:
addressed via b06181632f3e7609d6a71fd3b55fa0b4fe5fc800